code/modules/projectiles/projectile/bullets.dm

```diff
@@ -121,11 +121,12 @@
 /* short-casing projectiles, like the kind used in pistols or SMGs */
 
 /obj/item/projectile/bullet/pistol
-	damage = 25
-	armor_penetration = 10
+	damage = 20
+	armor_penetration = 15
 
 /obj/item/projectile/bullet/pistol/medium
 	damage = 30
+	armor_penetration = 0
 
 /obj/item/projectile/bullet/pistol/medium/ap
 	armor_penetration = 35
```

Comment:
consider nerfing this as well.

Reply:
Basically unused

Reply:
it's used for the mech SMG and will probably almost instantly kill anyone the few times its used.

Reply:
Good catch